.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
```

Comment:
We can use .25, there a PR ready about to be merged at any moment.
Can we use ERB in travis template? That would be cool to synchronize the version with this value https://github.com/ph/logstash/blob/263232a5e4b54c5fb286845bdc630ff494c9eb0a/rakelib/vendor.rake#L3